pinot-common/src/main/java/org/apache/pinot/pql/parsers/pql2/ast/SelectAstNode.java

```diff
@@ -31,12 +31,12 @@
  * AST node for a SELECT statement.
  */
 public class SelectAstNode extends BaseAstNode {
+  public static final int DEFAULT_RECORD_LIMIT = 10;
```

Comment:
Is it possible to just have one variable for TOP/LIMIT? Or does LIMIT imply that there may not be any sorting if ORDERBY not provided?

Reply:
Yes LIMIT implies no sorting/ordering unless ORDER BY is specified. In fact in MySql, both LIMIT and TOP are synonymous. 

I think we should eventually move to LIMIT -- now that we have GROUP BY with ORDER BY and are also converging towards standard SQL, we should have two variants (1) LIMIT and (2) LIMIT with ORDER BY

Reply:
Wondering what is a good use case for non ordered response? And/or if we should consider a default ordering?